### ISSUE TITLE
Observing the value of a picker as Ember computed property

### ISSUE THREAD

**Issue by rocketbop** [AddJam/ember-cli-pickadate#7]
I want to observe the collection date so I can update my options to the delivery date (ie 2 days after whichever collection date is selected.)

Is there a way to do this? I have tried variations on:

```
{{pick-a-time value=collectionDate date=date placeholder="Pick a time" options=opts}}
```

and creating a computed property on the controller:

```
  pickUpDate: Ember.computed(collectionDate, function() {
    this.get('collectionDate');
```

  }),

But I get collectionDate as undefined.

**Comment by chrissloey** [comment]
Try changing

`pickUpDate: Ember.computed(collectionDate, function() {`

to

`pickUpDate: Ember.computed('collectionDate', function() {`

**Comment by rocketbop** [comment]
Thanks. That doesn't seem to work though. Still undefined.

If I could even get my deliveryDateOptions object to recompute after the input for collection date is clicked, I could add in the necessary logic. This is what I have.

In the template, 

```
 {{pick-a-date value=collectionDate class="collection-date" placeholder="Collection date" options=collectionDateOptions}}
```

On my controller.

```
  deliveryDateOptions: Ember.computed('collectionDate', function () {
   // Calc options
  }}
```

I thought this would trigger once the input of pickadate changes, or the picker is closes, but it only triggers once, on page load. Well thanks anyway, I'm sure I'll figure it out :)

**Comment by chrissloey** [comment]
Had a play around with it - I think I fixed it in v0.3.1 :) let me know if it works for you now

**Comment by rocketbop** [comment]
It's working for me now! You're a star thank you so much :) Mucho appreciated.
